core/sql/sqlcomp/PrivMgrMD.h

```diff
@@ -56,35 +58,29 @@ typedef int             Lng32;
 // -----------------------------------------------------------------------
 class ObjectPrivsRow;
 class PrivMgrMDAdmin;
+class ObjectReference;
+class ObjectUsage;
 
 // -----------------------------------------------------------------------
 // Struct definitions
 // -----------------------------------------------------------------------
-typedef struct {
-  int64_t objectUID;
-  int32_t granteeID;
-  bool grantorIsSystem;
-  std::string objectName;
-  ComObjectType objectType;
-  PrivMgrDesc originalPrivs;
-  PrivMgrDesc updatedPrivs;
+struct ColumnReference {
+  int32_t columnOrdinal;
+  PrivMgrCoreDesc originalPrivs;
+  PrivMgrCoreDesc updatedPrivs;
+  ColumnReference() 
+  : columnOrdinal(-1),
+    originalPrivs(),
+    updatedPrivs(){};
 
   void describe (std::string &details) const
   {
-    details = "object usage - type is ";
-    char objectTypeLit[3] = {0};
-    strncpy(objectTypeLit,PrivMgr::ObjectEnumToLit(objectType),2);
-    details += objectTypeLit;
-    details += ", UID is ";
-    details += to_string((long long int) objectUID);
-    details += ", name is ";
-    details += objectName;
-    details += ", grantee is ";
-    details += to_string((long long int) granteeID);
-    details += ", is owner ";
-    details += (grantorIsSystem) ? "true " : "false ";
+    details = "column usage ";
+    details += " column number is ";
```

Comment:
You'll get two spaces between "column usage" and "column number is". Is that what you want?